Rdmp.Core/CommandLine/Interactive/ConsoleInputManager.cs

```diff
@@ -285,11 +283,13 @@ public override FileInfo SelectFile(string prompt, string patternDescription, st
 
             Console.WriteLine(prompt);
             var file = Console.ReadLine();
-            
-            if(file != null)
-                return new FileInfo(file);
 
-            return null;
+            if (string.IsNullOrWhiteSpace(file)) return null;
+            if (file.Equals("null", StringComparison.CurrentCultureIgnoreCase))
+                return new FileInfo(System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
```

Comment:
I think this should just return `null`.  That is consistent with `CommandLineObjectPicker`.  We have a lot of other bits of code that are:

- Pick a thing
- Or pick null
- Or cancel

For example "I want to change the extraction pk field but maybe I want to clear it (set it to null)" .  So I think we shouldn't complicate things here and just return null.

It should be pretty easy for users to write `/dev/null` if they really want.